snippets/base/urls.py

```diff
@@ -18,7 +18,6 @@
     url(r'^preview/$', views.preview_snippet, name='base.preview'),
     url(r'^show/(?P<snippet_id>\d+)/$', views.show_snippet, name='base.show'),
     url(r'^json-snippets/', views.JSONSnippetIndexView.as_view(), name='base.index_json'),
-    url(r'^active-snippets.json', views.ActiveSnippetsView.as_view(), name='base.active_snippets'),
```

Comment:
is this just no longer in use?

Reply:
Yes, it was a request for a custom view back in the day. I crossed checked with the original consumers and they don't use it. We never advertised it either.